### ISSUE TITLE
Bug Révisions

### ISSUE THREAD

**Issue by touti** [QuartiersXXI/quartiersxxi#17]
Lorsque l'on visite un article dans l'espace privé, sans l'avoir même ouvert en édition, on se retrouve en révisions pour les images …

**Comment by rastapopougros** [comment]
Effectivement, je reproduis avec mon compte aussi. Pourtant pour que ça ajoute une révision il faut forcément que l'API d'édition d'objet ait été lancée, et je ne vois pas qu'est-ce qui ferait ça juste en allant dans la vue… Vraiment bizarre. (rôle document ? mais pourquoi si on a touché à rien ?)

**Comment by touti** [comment]
Testé en local, apparemment c'est le plugin role qui a un souci dans cette version. Avec la version à jour (1.6.2) http://zone.spip.org/trac/spip-zone/browser/_plugins_/roles/branches/v1 le problème semble disparaitre.

**Comment by touti** [comment]
Raaa non, toujours pas résolu malgré la MAJ du plugin !
